config/manager/kustomization.yaml

```diff
@@ -14,4 +14,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: gcr.io/k8s-staging-kueue/kueue
-  newTag: main
+  newTag: 0.2.0-devel-159-g59bf53b-dirty
```

Comment:
revert (maybe you got a failure when trying to deploy the image)

Reply:
changed by one of the scripts probably